static/app/views/issueDetails/traceTimeline/traceTimelineTooltip.tsx

```diff
@@ -20,13 +20,6 @@ interface TraceTimelineTooltipProps {
 
 export function TraceTimelineTooltip({event, timelineEvents}: TraceTimelineTooltipProps) {
   const organization = useOrganization();
-  const location = useLocation();
-  const {projects} = useProjects({
-    slugs: [
-      ...timelineEvents.reduce((acc, cur) => acc.add(cur.project), new Set<string>()),
-    ],
-    orgId: organization.slug,
-  });
```

Comment:
This and the following removed lines are moved into the new `EventItem` component.